lib/l10n/app_en.arb

```diff
@@ -118,14 +118,14 @@
   "two_fa_failure_authenticator_title": "Authenticator not available",
   "two_fa_failure_authenticator_description": "To set up an Authenticator app, please first link an email address or phone number to your account",
   "two_fa_warning": "If you delete two-step authentication, it may put the security of your data at risk.",
-  "sign_up_passkey_title": "Passkeys are a better way to sign in",
+  "sign_up_passkey_title": "Register with passkey",
   "sign_up_passkey_advantage_1_title": "No password to remember",
   "sign_up_passkey_advantage_1_description": "With passkey, you can use things like your fingerprint or face to login",
   "sign_up_passkey_advantage_2_title": "Works on all of your devices",
   "sign_up_passkey_advantage_2_description": "Passkey will automatically be available across your synced devices",
   "sign_up_passkey_advantage_3_title": "Keep your account safer",
   "sign_up_passkey_advantage_3_description": "Passkey offer state-of-the-art phishing resistance",
-  "sign_up_passkey_button": "Continue with passkey",
+  "sign_up_passkey_button": "Continue",
```

Comment:
we may use `button_continue` instead of declaring a duplicate, wdyt?

Reply:
Done